scripts/accessibility/tidy_agent_data.R

```diff
@@ -58,14 +58,21 @@ tours_1 <- tours_1 %>%
 
 # Join tours to agents data ----
 
+res_var <- c(
+  "total_access",
+  "sustainable_access",
+  "gen_cost",
+  "cost"
```

Comment:
`gen_cost` and `cost` are not available in my result data. Are they provided in newer versions of Helmet?

Reply:
Yep, sorry I forgot to update input data. These files are added in Helmet Pull [323](https://github.com/HSLdevcom/helmet-model-system/pull/323). Uploaded these to Sharepoint data folder.